src/main/java/com/azure/cosmos/examples/modeling/async/Deployment.java

```diff
@@ -0,0 +1,310 @@
+                .endpoint(ChangeFeedConfigurations.HOST)
+                .key(ChangeFeedConfigurations.MASTER_KEY)
+                .contentResponseOnWriteEnabled(true)
+                .consistencyLevel(ConsistencyLevel.SESSION)
+                .buildAsyncClient();
+    }
+
+    public void LoadContainersFromFolder(int schemaVersion, String SourceDatabaseName,
+                                         String TargetDatabaseName) {
+        List<List<SchemaDetails>> DatabaseSchema = getSchemaDetails();    
+        CosmosAsyncClient clientAsync = getCosmosClient();
+        CosmosAsyncDatabase database = clientAsync.getDatabase(TargetDatabaseName);
+        Path currentRelativePath = Paths.get("");
+        String s = currentRelativePath.toAbsolutePath().toString();
+        String folder = s + "/src/main/java/com/azure/cosmos/examples/data" + "/" + SourceDatabaseName + "/";
+        folder = folder.replace("\\", "/");
+        System.out.println("folder: " + folder);
+        System.out.println("Preparing to load containers and data for " + TargetDatabaseName + "....");
+        File path = new java.io.File(folder);
+        File[] listOfFiles = path.listFiles();
+        final ExecutorService es = Executors.newCachedThreadPool();
+
+        assert listOfFiles != null;
+        for (File file : listOfFiles) {
+            final Runnable task = () -> {
+                System.out.println("new container thread...");
+                if (file.isFile()) {
+                    String pk = "";
+                    System.out.println("loading data for container: " + file.getName()+" in database "+TargetDatabaseName);
+                    System.out.println("schemaVersion: "+schemaVersion);
+                    List<SchemaDetails> schemaDetails = DatabaseSchema.get(schemaVersion -1);
+                    for (SchemaDetails schema : schemaDetails) {
+                        if (file.getName().equals(schema.getContainerName())) {
+                            pk = schema.getPk().substring(1);
+                            try {
+                                Thread.sleep(0);
+                            } catch (InterruptedException e) {
+                                e.printStackTrace();
+                            }
+                        }
+                    }
+                    Scanner sc;
+                    try {
+                        sc = new Scanner(file);
+                        sc.useDelimiter("\\Z");
+                        String JsonArrayString = sc.next();
+                        JSONArray jsonArray = new JSONArray(JsonArrayString);
+                        List<JsonNode> docList = new ArrayList<>();
+                        for (Object doc : jsonArray) {
+                            ObjectMapper mapper = new ObjectMapper();
+                            JsonNode actualObj = mapper.readTree(doc.toString());
+                            docList.add(actualObj);
+                            Thread.sleep(0);
```

Comment:
why the thread sleep 0 here do?

Reply:
or below

Reply:
also e.printStackTrace is frowned upon - forces the output - using logger is definitely better

Reply:
Thanks, will change